challenge_4/go/erocs/invert_tree.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+)
+
+type TreeNode struct {
+	Left  *TreeNode
+	Right *TreeNode
+	Value int
+}
+
+func (t *TreeNode) String() string {
+	l := ""
```

Comment:
I think you'd more typically see 
```go
var l string
var r string
```

You could do the same for the numbers as above but it's mainly in strings where it's a bit odd to use shorthand to create an empty value string when it's already that way with `var`